hasjob/models/filterset.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+
+from flask import url_for
+from sqlalchemy.dialects import postgresql
+from sqlalchemy import event, DDL
+from . import db, BaseScopedNameMixin, JobType, JobCategory, Tag, Domain, Board
+from ..extapi import location_geodata
+
+
+__all__ = ['FilterSet']
```

Comment:
Just noticed this. Per naming convention this should be `Filterset` or the table should be `filter_set`. (I know `JobPost` is a glaring example, but it predates good convention and should be renamed to `Jobpost`.)